src/test/java/com/charity/charitydonationstracker/CharityDonationTests.java

```diff
@@ -0,0 +1,96 @@
+package com.charity.charitydonationstracker;
+
+import com.charity.charitydonationstracker.entity.CharityDonation;
+import com.charity.charitydonationstracker.repository.CharityDonationsRepository;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.Year;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.springframework.test.util.AssertionErrors.assertTrue;
+
+@SpringBootTest
+public class CharityDonationTests {
+    @Autowired
+    private CharityDonationsRepository charityDonationsRepository;
+
+    private List<CharityDonation> donations;
+    
+    @BeforeAll
+    static void initAll() {
+    }
+
+    @BeforeEach
+    void init() {
+        donations = charityDonationsRepository.findAll();
+    }
+
+    @Test
+    public void charityDonationsCountTest() {
+        List<CharityDonation> donations = charityDonationsRepository.findAll();
+         int charityDonationsCountTest = donations.size();
+         int expectedValue = 10;
+        assertTrue("Charity donations count is greater than " + expectedValue, charityDonationsCountTest > expectedValue);
+    }
+
+    @Test
+    public void financialYearWiseCount() {
+        Map<String, Integer> yearCounts = new HashMap<>();
+        for (CharityDonation donation: donations) {
+            String year = donation.getFinancialYear();
+            yearCounts.put(year, yearCounts.getOrDefault(year, 0)+1);
+        }
+
```

Comment:
The test method is not testing anything, instead it is just printing something on the console.

Reply:
Incorporated the comments.